shell/src/main/java/tachyon/shell/TFsShellUtils.java

```diff
@@ -74,4 +80,163 @@ public static String validatePath(String path, TachyonConf tachyonConf) throws I
       return PathUtils.concatPath(Constants.HEADER + hostname + ":" + port, path);
     }
   }
+  
+  /**
+   * Get all the TachyonURIs that match inputURI. 
+   * If the path is an absolute path, the returned list only contains the corresponding URI;
+   * Else if the path contains wildcards, the returned list contains all the matched URIs
+   * It supports any number of wildcards in inputURI
+   * @param tachyonClient The client that fetches the paths
+   * @param inputURI the input URI (could contain wildcards)
+   * @return A list of TachyonURI that matches the inputURI
+   * @throws IOException
+   */
+  public static List<TachyonURI> getTachyonURIs(TachyonFS tachyonClient, TachyonURI inputURI)
+      throws IOException {    
+    if (inputURI.getPath().contains(TachyonURI.WILDCARD) == false) {
```

Comment:
A better style would be `if (!inputURI.getPath().contains(TachyonURI.WILDCARD))`